src/react_native/gesture.cljs

```diff
@@ -91,7 +91,7 @@
     [flat-list
      (merge props
             {:data      data
-             :render-fn (fn [item]
-                          (if (:header? item)
-                            (render-section-header-fn item)
-                            (render-fn item)))})]))
+             :render-fn (fn [p1 p2 p3 p4]
```

Comment:
it doesn't matter for this function, we don't care about params we just need to pass them

Reply:
I see, still a bit weird to me, but it's a minor detail for sure. If this part of the code didn't care, it could just grab the arguments and pass them along without giving any name, thus giving a stronger sense of "I don't care".

Okay, maybe my suggestion is not really an improvement in terms of clarity, but I'm sharing anyways ;) Up to you to judge of course.

```clojure
(defn section-list
  [{:keys [sections render-section-header-fn render-fn] :as props}]
  (let [data (flatten-sections sections)]
    [flat-list
     (merge props
            {:data      data
             :render-fn (fn [arg & more]
                          (if (:header? arg)
                            (into [render-section-header-fn arg] more)
                            (into [render-fn arg] more)))})]))
```